contrib/docker/docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
     privileged: true
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - /var/run/netns:/host/run:shared
+      - /run/netns:/host/run:shared
```

Comment:
Sent an e-mail ... something still doesn't work for me even with /run/netns ... I think that the problem is with :shared tag